src/test_typing.py

```diff
@@ -2057,8 +2051,14 @@ class NonDefaultAfterDefault(NamedTuple):
     def test_annotation_usage_with_methods(self):
         self.assertEquals(XMeth(1).double(), 2)
         self.assertEquals(XMeth(42).x, XMeth(42)[0])
-        self.assertEquals(XMethBad(1)._fields, ('x',))
-        self.assertEquals(XMethBad(1).__annotations__, {'x': int})
+
+        with self.assertRaises(AttributeError):
```

Comment:
Can you also add a test where a method name conflicts with a NamedTuple fields? `_fields` seems like a slightly different case.